packages/fast-components-class-name-contracts-msft/src/managed-classes.ts

```diff
@@ -0,0 +1 @@
+export * from "@microsoft/fast-components-class-name-contracts-base";

```

Comment:
Are we sure we don't need to import this?

Reply:
@nicholasrice I don't think we do and if I try I get 'Circular definition of import' error

Reply:
You do not need to import it